relay-server/src/services/store.rs

```diff
@@ -812,6 +834,63 @@ impl StoreService {
         Ok(())
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
not sure what to do about the too many args here

Reply:
Instead of passing `replay_event`, `replay_recording`, and a boolean flag, the most idiomatic would probably be to pass an enum like 

```rust
enum Data {
    Event(&Item),
    Recording(&Item),
    Combined { event: &Item, recording: &Item }
}

That would also make states like `(None, None, true)` impossible.